bin/i18n/resources/dashboard/course_content.rb

```diff
@@ -7,8 +8,8 @@ module Resources
     module Dashboard
       module CourseContent
         DIR_NAME = 'course_content'.freeze
```

Comment:
Super curious about this: If there are two `DIR_NAME` variables (one with 'dashboard' and one with 'course_content'). Does `DIR_NAME` keep the value assigned in the current Module (or the closes parent module), or is there a possibility for conflict?

Reply:
There are no conflicts since `I18n::Resources::Dasboard` and `I18n::Resources::Dasboard::CourseContent` are separate independent scopes

Reply:
`I18n::Resources::Dasboard::DIR_NAME` != `I18n::Resources::Dasboard::CourseConten::DIR_NAME`

Reply:
And what is the default scope?